src/consts.ts

```diff
@@ -71,10 +71,13 @@ export enum DeviceType {
     DUPLO_TRAIN_BASE_SPEEDOMETER = 44,
     CONTROL_PLUS_LARGE_MOTOR = 46,
     CONTROL_PLUS_XLARGE_MOTOR = 47,
+    CONTROL_PLUS_GEST = 54,
```

Comment:
I checked the values returned by this sensor. This seems to be a value in the range of 0-4 generated from the motion sensors. Most likely GEST stands for "gesture". Shaking gives 3. Tossing up - 4 when raising and 1 or 2 when landing. 0 when no movement.